sdk/samples/radio_example_rx/main.c

```diff
@@ -7,19 +7,44 @@
 #include "radio.h"
 #include "rftimer.h"
 #include "scm3c_hw_interface.h"
+#include "scum.h"
 
-// Number of for loop cycles between Hello World messages.
-// 700000 for loop cycles roughly correspond to 1 second.
-#define NUM_CYCLES_BETWEEN_TX (1000000UL)
+static uint8_t packet[125] = {0};
```

Comment:
Also all changes in samples/ should be part of their own PR if they make sense and are supposed to be contributed upstream